addons/cni-canal/canal.yaml

```diff
@@ -4474,9 +4489,18 @@ spec:
             # Set Felix endpoint to host default action to ACCEPT.
             - name: FELIX_DEFAULTENDPOINTTOHOSTACTION
               value: "ACCEPT"
+            {{ if .Config.ClusterNetwork.HasIPv6 }}
```

Comment:
You have `CALICO_IPV4POOL_CIDR` on L4484 which should be handled in the same way as for Calico VXLAN.